cmd/kubeadm/app/phases/kubelet/config.go

```diff
@@ -203,6 +221,22 @@ func writeConfigBytesToDisk(b []byte, kubeletDir string) error {
 	return nil
 }
 
+// writeInstanceConfigBytesToDisk writes a byte slice down to disk at the specific location of the Instance kubelet config file
+func writeInstanceConfigBytesToDisk(b []byte, kubeletDir string) error {
```

Comment:
instead of adding a new function you should modify the existing function writeConfigBytesToDisk
move the fmt.Printf() outside of it on the caller side.

so use the same function for both the regular and instance-config
make the error strings more generic:

```
errors.Wrpf(err, failed to write kubelet configuration file %q)
```